server/api/crud/model_monitoring/deployment.py

```diff
@@ -906,12 +931,40 @@ async def delete_monitoring_function(
                     v3io_client.stream.delete(
                         container, stream_path, access_key=access_key
                     )
+                    logger.debug("Deleted v3io stream", stream_path=stream_path)
                 except v3io.dataplane.response.HttpResponseError as e:
                     logger.warning(
-                        f"Can't delete {function_name}'s stream",
+                        "Can't delete v3io stream",
                         stream_path=stream_path,
                         error=e,
                     )
+        elif stream_paths[0].startswith("kafka://"):
+            # Delete Kafka topics
+            import kafka
+            import kafka.errors
+
+            topics = []
+
+            topic, brokers = mlrun.datastore.utils.parse_kafka_url(url=stream_paths[0])
+            topics.append(topic)
+
+            for stream_path in stream_paths[1:]:
+                topic, _ = mlrun.datastore.utils.parse_kafka_url(url=stream_path)
+                topics.append(topic)
+
+            try:
+                kafka_client = kafka.KafkaAdminClient(
+                    bootstrap_servers=brokers,
+                    client_id=f"{project}",
```

Comment:
This f-string is unnecessary:
```suggestion
                    client_id=project,
```